w_common/lib/src/common/disposable.dart

```diff
@@ -122,7 +122,7 @@ class _ObservableTimer implements Timer {
 
 /// A class used as a marker for potential memory leaks.
 class LeakFlag {
-  final String description;
+  final String? description;
 
   LeakFlag(this.description);
```

Comment:
#nit not sure it matters much, but since `description` is nullable it might make sense for this parameter to be optional.